src/main/java/com/edu/unq/tpi/dapp/grupoB/Eventeando/dominio/PotluckEvent.java

```diff
@@ -0,0 +1,52 @@
+package com.edu.unq.tpi.dapp.grupoB.Eventeando.dominio;
+
+import com.edu.unq.tpi.dapp.grupoB.Eventeando.exceptions.EventException;
+import com.edu.unq.tpi.dapp.grupoB.Eventeando.validators.EventValidator;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class PotluckEvent extends Event {
+
+
+    protected HashMap<String, User> coveredExpenses = new HashMap<>();
+
+    public List<String> coveredExpenses() {
+        return new ArrayList<>(this.coveredExpenses.keySet());
+    }
+
+    public void aGuestTakeChargeOf(User guest, String anExpense) {
+        validateThatTheUserWasInvited(guest.email());
+        validateExistanceOfExpense(anExpense);
+        validateThatExpenseIsNotAlreadyCovered(anExpense);
+        this.coveredExpenses.put(anExpense, guest);
```

Comment:
Es mejor poner los `this`? Pregunto por ignorancia, yo no lo hago :see_no_evil:

Reply:
Hay gente que le gusta y gente que no jajaja no cambia nada.